scipy/optimize/_dual_annealing.py

```diff
@@ -495,6 +491,8 @@ def dual_annealing(func, x0, bounds, args=(), maxiter=1000,
             - 2: detection done in the dual annealing process.
 
         If the callback implementation returns True, the algorithm will stop.
+    x0 : ndarray, shape(n,), optional
+        A single initial starting point coordinates.
```

Comment:
Plural "coordinates" doesn't sound right here?

Reply:
@tylerjereddy, I was meaning coordinates of a single starting point (like a n-dimension starting point), do you think it has to be "coordinate". I am not a native English speaker, so I may wrote this wrongly...

Reply:
Maybe something similar to your suggestion then -- "Coordinates of a single n-dimensional starting point"?

Reply:
Thanks a lot! Sounds good. Adjusted.